.travis.yml

```diff
@@ -77,19 +77,19 @@ jobs:
       script:
         - unbuffer node build-system/pr-check/remote-tests.js
       after_script:
-        - build-system/sauce_connect/stop_sauce_connect.sh
+        - ps -ef
     - stage: test
       name: "End to End Tests"
       script:
         - unbuffer node build-system/pr-check/e2e-tests.js
+      after_script:
+        - ps -ef
 cache:
   directories:
     - node_modules
     - build-system/tasks/e2e/node_modules
     - build-system/tasks/visual-diff/node_modules
-    - sauce_connect
     - validator/node_modules
     - validator/nodejs/node_modules
     - validator/webui/node_modules
-    - $HOME/.cache/pip
```

Comment:
Travis' native `pip` caching [only works](https://docs.travis-ci.com/user/caching#pip-cache) for python projects. `$HOME/.cache/pip` was cached based on [these](https://github.com/nickstenning/travis-pip-cache) results.

From the logs, [this line](https://travis-ci.org/ampproject/amphtml/jobs/547876254#L296) suggests that a cached install location is being used (even though, as you pointed out, the cache is [updated](https://travis-ci.org/ampproject/amphtml/jobs/547876254#L2097-L2100) at the end of the build). Curious: does the install time stay the same when caching is disabled?

Reply:
Interesting repo! By removing it from the cache, `pip install` increases from 3s to 5s but `store build build` decreases from 85s to 10s.

Reply:
Yeah, good catch on your part in noticing the much slower `store` step!